samples/IdentitySample.Mvc/Startup.cs

```diff
@@ -47,21 +47,6 @@ public void ConfigureServices(IServiceCollection services)
                     .AddEntityFrameworkStores<ApplicationDbContext>()
```

Comment:
I still need to go and fix the security middleware Configure methods to be called Add